esp/src/src-react/components/LogicalFileSummary.tsx

```diff
@@ -201,8 +201,8 @@ export const LogicalFileSummary: React.FunctionComponent<LogicalFileSummaryProps
                 "RecordCount": { label: nlsHPCC.RecordCount, type: "string", value: file?.RecordCount, readonly: true },
                 "IsReplicated": { label: nlsHPCC.IsReplicated, type: "checkbox", value: (file?.filePartsOnCluster() ?? []).length > 0, readonly: true },
                 "NumParts": { label: nlsHPCC.FileParts, type: "number", value: file?.NumParts, readonly: true },
-                "MinSkew": { label: nlsHPCC.MinSkew, type: "string", value: `${Utility.formatDecimal(file?.Stat?.MinSkewInt64 / 100 ?? 0)}%`, readonly: true },
-                "MaxSkew": { label: nlsHPCC.MaxSkew, type: "string", value: `${Utility.formatDecimal(file?.Stat?.MaxSkewInt64 / 100 ?? 0)}%`, readonly: true },
+                "MinSkew": { label: nlsHPCC.MinSkew, type: "string", value: `${Utility.formatDecimal((file?.Stat?.MinSkewInt64 ?? 0) / 100)}%`, readonly: true },
```

Comment:
Not relevant to this review really. But I could've sworn you'd made this change already? Kunal has a PR with these deleted lines `... / 100 ?? 0)}` in it as well and I was confused why it was there.

Reply:
This an old PR rebased and updated...